modules/features/fe_idea/fe_idea.features.user_permission.inc

```diff
@@ -51,6 +51,7 @@ function fe_idea_user_default_permissions() {
   $permissions['create idea content'] = array(
     'name' => 'create idea content',
     'roles' => array(
+      'administrator' => 'administrator',
```

Comment:
`authenticated user` is enough. Why do you add `administrator` as well?

Reply:
this is automatic, i'm don't add this.

Reply:
You should go over the diff before you commit and verify it's valid. If needed - edit and recommit